angular/src/services/jslib-services.module.ts

```diff
@@ -0,0 +1,416 @@
+                PolicyServiceAbstraction,
+                SendServiceAbstraction,
+                LogService,
+                TokenServiceAbstraction,
+                KeyConnectorServiceAbstraction,
+            ],
+        },
+        {
+            provide: UserServiceAbstraction,
+            useClass: UserService,
+            deps: [TokenServiceAbstraction, StorageServiceAbstraction],
+        },
+        { provide: BroadcasterServiceAbstraction, useClass: BroadcasterService },
+        {
+            provide: SettingsServiceAbstraction,
+            useClass: SettingsService,
+            deps: [UserServiceAbstraction, StorageServiceAbstraction],
+        },
+        {
+            provide: VaultTimeoutServiceAbstraction,
+            useFactory: (cipherService: CipherServiceAbstraction, folderService: FolderServiceAbstraction,
+                collectionService: CollectionServiceAbstraction, cryptoService: CryptoServiceAbstraction,
+                platformUtilsService: PlatformUtilsServiceAbstraction, storageService: StorageServiceAbstraction,
+                messagingService: MessagingServiceAbstraction, searchService: SearchServiceAbstraction,
+                userService: UserServiceAbstraction, tokenService: TokenServiceAbstraction,
+                policyService: PolicyServiceAbstraction, keyConnectorService: KeyConnectorServiceAbstraction) =>
+                new VaultTimeoutService(cipherService, folderService, collectionService, cryptoService,
+                    platformUtilsService, storageService, messagingService, searchService, userService, tokenService,
+                    policyService, keyConnectorService, null,
+                    async () => messagingService.send('logout', { expired: false })),
+            deps: [
+                CipherServiceAbstraction,
+                FolderServiceAbstraction,
+                CollectionServiceAbstraction,
+                CryptoServiceAbstraction,
+                PlatformUtilsServiceAbstraction,
+                StorageServiceAbstraction,
+                MessagingServiceAbstraction,
+                SearchServiceAbstraction,
+                UserServiceAbstraction,
+                TokenServiceAbstraction,
+                PolicyServiceAbstraction,
+            ],
+        },
+        { provide: StateServiceAbstraction, useClass: StateService },
+        {
+            provide: ExportServiceAbstraction,
+            useClass: ExportService,
+            deps: [
+                FolderServiceAbstraction,
+                CipherServiceAbstraction,
+                ApiServiceAbstraction,
+                CryptoServiceAbstraction,
+            ],
+        },
+        {
+            provide: SearchServiceAbstraction,
+            useClass: SearchService,
+            deps: [
+                CipherServiceAbstraction,
+                LogService,
+                I18nServiceAbstraction,
+            ],
+        },
+        {
+            provide: ImportServiceAbstraction,
```

Comment:
You can only import in the web vault, I think this should be omitted from the jslib module and provided for in the client's `services.module.ts` instead.

Reply:
Should we also look into moving the code to `web`? To minimize the need to jump between the repos.

Reply:
I would be okay with that, I assume that it's in jslib because import & export logically go together, but I think we should be stricter with only including _shared_ code in jslib.

Reply:
you can import in CLI too

Reply:
Ah good point @MGibson1. Then we have to leave it in jslib. I would still exclude it from the exported module though.